CODE_OF_CONDUCT.md

```diff
@@ -59,7 +59,7 @@ representative at an online or offline event.
 ## Enforcement
 
 Instances of abusive, harassing, or otherwise unacceptable behavior may be
-reported to the community leaders responsible for enforcement at bigrails@gusto.com.
+reported to the community leaders responsible for enforcement at rubyatscale@gusto.com.
```

Comment:
We did have a chat with the Rails team members while at RailsConf. They expressed their worry that because it wasn’t an official project ("yet" anyways 😄), we might confuse folks. The name change was then pretty easy though, because we realized that almost all of our tooling applies to all Ruby apps … hence the new name!

Reply:
haha makes sense. I reeeally liked `bigrails` but it is what it is 😄